### ISSUE TITLE
LogiTrack

### ISSUE THREAD

**Issue by felixlinker** [felixlinker/W_Missionstemplate#10]
LogiTrack sollte automatisch initialisiert werden.

**Comment by DasCleverle** [comment]
Warte mit der Umsetzung noch ein bisschen. Ich überarbeite grad DLP, dann geht das einfacher.

**Comment by GitHawk** [comment]
@DasCleverle Hast du das nun umgesetzt?

**Comment by DasCleverle** [comment]
Ja, ist in Version 2.0.0 von DLP drinnen. Details dazu hier https://www.gruppe-w.de/forum/viewthread.php?thread_id=2744

**Comment by GitHawk** [comment]
Cool, dann kann das @felixlinker mal ins Template einbauen :-)

**Comment by felixlinker** [comment]
Ist die Version schon im Modset?

**Comment by GitHawk** [comment]
@elvaron kann das wohl beantworten

**Comment by DasCleverle** [comment]
Ja, ist drin. Du kannst direkt loslegen :)
